### PR TITLE
add our standard rails template for api based rails apps

### PR DESCRIPTION
**Pull request by soberstadt**

### CODE REVIEW

File: cruds_rails/.gitignore [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore the default SQLite database.
+/db/*.sqlite3
```
Comment (twinge) [inline cruds_rails/.gitignore]: The sqlite3 lines aren't needed

Comment (soberstadt) [inline cruds_rails/.gitignore]: fixed :heavy_check_mark:

File: cruds_rails/template.rb [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+run 'touch Gemfile'
+add_source 'https://rubygems.org'
+gem 'rails', '~> 4.2.1'
+gem 'puma'
+
+gem 'pg'
+
+gem 'newrelic_rpm'
+gem 'rack-cors', require: 'rack/cors'
+gem 'redis', '~> 3.2.1'
```
Comment (twinge) [inline cruds_rails/template.rb]: This will get brought in by redis-rails

File: cruds_rails/template.rb [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  gem 'dotenv-rails'
+  gem 'guard'
+  gem 'guard-rubocop'
+  gem 'rspec-rails'
+  gem 'spring'
+  gem 'pry-rails'
+end
+
+gem_group :test do
+  gem 'guard-rspec'
```
Comment (twinge) [inline cruds_rails/template.rb]: Why is guard-rspec in this group and the other 2 guard things in the other group?

File: cruds_rails/template.rb [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+gem 'rack-cors', require: 'rack/cors'
+gem 'redis', '~> 3.2.1'
+gem 'redis-rails'
+gem 'rollbar'
+gem 'silencer'
+gem 'syslog-logger'
+
+gem_group :development, :test do
+  gem 'dotenv-rails'
+  gem 'guard'
```
Comment (twinge) [inline cruds_rails/template.rb]: This will get brought in as a dependency by guard-rubocop or guard-rspec

File: cruds_rails/template.rb [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  gem 'spring'
+  gem 'pry-rails'
+end
+
+gem_group :test do
+  gem 'guard-rspec'
+  gem 'webmock'
+  gem 'simplecov', require: false
+  gem 'factory_girl_rails'
+  gem 'shoulda', require: false
```
Comment (twinge) [inline cruds_rails/template.rb]: I don't think most apps use shoulda for anything. Maybe they shoulda?

**Comment by soberstadt** [comment]
:heavy_check_mark: fixed

**Pull request merged**